DigitalLearningSolutions.Web/Startup.cs

```diff
@@ -169,7 +169,9 @@ public void Configure(IApplicationBuilder app, IMigrationRunner migrationRunner)
             app.UseSession();
 
             app.UseEndpoints(endpoints =>
-                endpoints.MapControllerRoute("default", "{controller=Home}/{action=Index}"));
+            {
+                endpoints.MapControllerRoute("default", "{controller=Home}/{action=Index}");
+            });

```

Comment:
Is there a reason for this change?

Reply:
Leftover from when I was trying to figure out the routes, just forgot to change it back.